citi-onboarding/lgbtqia
# Feature/terms

## What I did

- create cards, carousel and info components
- create terms page
- import necessary css in App.tsx
- add Terms tag to App.tsx
- link Rubik font on client/public/index.html

## How to test

0. If you don't have this repository in your computer:
   1. create a folder
   2. In terminal
	   1. find your folder
	   2. `git clone https://github.com/citi-onboarding/lgbtqia.git`

1. Open the terminal and go to the folder of this project
2. In terminal:
   1. `git fetch`
   2. `git checkout feature/terms`
   3. open the client folder and enter `yarn install` and then `yarn start`
4. Test the terms page on your browser and try changing the size of the window
5. Go through the carousel and see if there are any problems
6. Try pinching the window and tell me how it goes (I wasn't able to test it on my device, because I use linux)
7. Test the things that you think are worth testing, even the ones that are not in this description
